src/Microsoft.ML.StandardTrainers/Standard/MultiClass/MetaMulticlassTrainer.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.ML.Trainers
 {
     using TScalarTrainer = ITrainerEstimator<ISingleFeaturePredictionTransformer<IPredictorProducing<float>>, IPredictorProducing<float>>;
 
-    public abstract class MetaMulticlassTrainer<TTransformer, TModel> : ITrainerEstimator<TTransformer, TModel>, ITrainer<IPredictor>
+    public abstract class MetaTrainer<TTransformer, TModel> : ITrainerEstimator<TTransformer, TModel>, ITrainer<IPredictor>
```

Comment:
Here's a special case of our rule:

> {TypeOfTask} is added only only when the algorithm supports multiple kinds of tasks 

I don't believe `MetaTrainer` is a good name here since it too general. That name makes it sound like it can train anything. I think we should keep "multiclass" in the name.  So probably `MetaMulticlassClassificationTrainer`.

Reply:
Yeah. It makes sense to treat this as a special case.